addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -681,25 +748,23 @@ registry.mediaVideo = publicWidget.Widget.extend({
         var m = src.match(/^(?:https?:)?\/\/([^/?#]+)/);
         if (!m) {
             // Unsupported protocol or wrong URL format, don't inject iframe
-            return def;
+            return undefined;
```

Comment:
`return undefined` feels strange to write. Either `return;` or return something `null` (which is of object type).